integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -121,6 +123,8 @@ spec:
           envFrom:
           - configMapRef:
               name: {{ template "alluxio.fullname" . }}-config
+{{ include "alluxio.worker.readinessProbe" . | indent 10 }}
```

Comment:
why is this not the job worker probe?

Reply:
I think the criteria for readiness is the same for worker and job-worker now so i chose to use the same probes for them both. And thus the "worker" has the same meaning with the directory "worker/", meaning the worker pod.

I can of course define the probes for worker and job-worker separately. But I don't think what we use for the probe will change, at least in the near future? What do you think? @madanadit

Reply:
@jiacheliu3 I don't get it. why is the criteria the same? the readiness / liveness probe is a per container concept not per pod. the probe for job worker should use `alluxio-monitor.sh` with arg `job_worker` to ping the job worker rpc?

Reply:
Sorry I was mistaken. The probes should be different for worker and job-worker, namely `alluxio-monitor.sh worker` and `alluxio-monitor.sh job_worker`. This has been corrected.